### PR TITLE
Completed readme

### PR DESCRIPTION
**Pull request by olivershh**

### CODE REVIEW

**Comment by olivershh** [comment]
I'm not sure whether it's too vague, though I realize it's audience is going to be tech savvy so didn't want to oversimplify.

**Comment by olivershh** [comment]
I'm just adding postgres and node requirements now.

**Review by popatre — CHANGES_REQUESTED** [review]
I would just make some adjustments. 
Don't presume anyone is tech savvy. I'd make your readme as helpful as possible for all. 

Think back to docs you've looked at that have presumed knowledge and remember how annoying they have been to try and use/get something working

File: package-lock.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
         "jest-sorted": "^1.0.14",
         "nodemon": "^2.0.19",
         "pg": "^8.7.3",
+        "pg-format": "^1.0.4",
         "supertest": "^6.2.4"
```
[the remark this answers is not in the document]
Comment (popatre) [reply package-lock.json:18]: In the package.json, not the lock file

File: README.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
-# Northcoders News API
+# Streamlined news API
 
-## Local Setup
+A simple API of news articles and comments. Check the homepage on the link below for a directory of endpoints and functionality.
 
-You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=<database_name_here>, with the correct database name for that environment (see /db/setup.sql for the database names).
+## Take a look
+
```
Comment (popatre) [inline README.md:6]: Needs a minimum version of node and postgres

**Review by popatre — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (olivershh) [reply package-lock.json:18]: done

File: README.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+https://nc-news-api-oliver.herokuapp.com/
+
```
[the remark this answers is not in the document]
Comment (olivershh) [reply README.md:10]: Added. I've actually linked the github instructions as they're very clear and it avoids cluttering up the readme but happy to change.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+## Setup Instructions
+
+`Installing dependencies`
+
+From your repo's directory in the command line, run the following command:
+
+```
+npm install -D
```
[the remark this answers is not in the document]
Comment (olivershh) [reply README.md]: Changed, thanks

File: README.md [hunk 5]
```diff
@@ -1,5 +1,62 @@
...
+
+```
+npm test
+```
+
+`Setting environment variables`
+
+You will need to create two .env files for your project: .env.test and .env.development. Into each, add the following:
+
+#### .env.test
```
[the remark this answers is not in the document]
Comment (olivershh) [reply README.md]: Good spot. Moved to just below cloning.

**Review by popatre — APPROVED** [review]

**Pull request merged**